src/main/java/at/hannibal2/skyhanni/events/EntityRenderNametagEvent.kt

```diff
@@ -0,0 +1,8 @@
+package at.hannibal2.skyhanni.events
+
+import net.minecraft.entity.Entity
+import net.minecraft.util.IChatComponent
+import net.minecraftforge.fml.common.eventhandler.Cancelable
+
+@Cancelable
+class EntityRenderNametagEvent(val entity: Entity, var chatComponent: IChatComponent) : LorenzEvent()
```

Comment:
Cancelable is not needed as you never change operation depending on cancel.

The event should be named something along the lines of `EntityDisplayNameEvent`

The event should be put into the `entity` package of `events`